tests/test_aws_auth.py

```diff
@@ -60,12 +62,24 @@ def test_get_token(self, mock_post):
             "renewable": True
         }
 
+        iam_info_response_body = {
+            "Code" : "Success",
+            "LastUpdated" : "2017-11-29T21:11:45Z",
+            "InstanceProfileArn" : "arn:aws:iam::123:instance-profile/rolepath/rolename",
+            "InstanceProfileId" : "AIPAJURSEGIIGKUJSHNXY"
+        }
+
+        iam_security_credentials_response_body = "rolename"
```

Comment:
If you want to make the test more accurate, the security response body can have a slightly different rolename than what is found in the InstanceProfileArn.  This happens sometimes when both are created with CloudFormation.

Reply:
To emphasize this, `the instance profile name` and `role name` as rarely ever the same, and are always different when `either` the IAM role or Instance profile is created via CloudFormation.

Reply:
I understand, I will make sure they are different to make test better.